tests/Dfe.PlanTech.Web.E2ETests.Beta/features/step_definitions/common.steps.ts

```diff
@@ -100,7 +100,7 @@ Then('I should see a back link to {string}', async function (href: string) {
   expect(actualHref).toContain(href);
 });
 
-Then('I should see a section heading with text {string}', async function (heading: string) {
+Then('I should see a h2 section heading with text {string}', async function (heading: string) {
   const headingLocator = this.page.locator(`h2.govuk-heading-l:has-text("${heading}")`);
```

Comment:
Non-blocking observation: Not every `<h2>` will _necessarily_ be `govuk-heading-l`.

I'm not familiar enough with the service to know/recognise if this matters _here_ though.

> For [Question pages in your service](https://design-system.service.gov.uk/patterns/question-pages/), or pages with long headings, follow the usual hierarchy of heading levels and styles associated with them. For example, `govuk-heading-l` for an `<h1>`, followed by `govuk-heading-m` for an `<h2>` and so on.

https://design-system.service.gov.uk/styles/headings/